prepack.js

```diff
@@ -1,34 +1,11 @@
 #!/usr/bin/env node
 'use strict';
 
-// add this check because withFileTypes was added in 10.10.0. for simplicity
-// just require node 12.
-if (process.versions.node.split('.').shift() < 12) {
```

Comment:
have we excluded Node 10.9 and earlier somewhere else
the code using withFileTypes is still on line 89, 94 ...

Reply:
The script runs as part of the publish process. It guards from human error, which in this case was publishing from lower version of node (that does not support the `withFileTypes` settings). Moving forward publish will always run on GH Actions, thus error not possible and check not needed.